python/hail/methods/impex.py

```diff
@@ -635,6 +741,24 @@ def import_plink(bed, bim, fam, min_partitions=None, delimiter='\\\\s+',
            drop_cols=bool,
            drop_rows=bool)
 def read_matrix(path, drop_cols=False, drop_rows=False):
+    """Read a `.vds` file as a :class:`.MatrixTable`
+
+    Parameters
+    ----------
+    path : :obj:`str`
+        File to read.
+
+    drop_samples : :obj:`bool`
```

Comment:
This should all be generic now.  drop_rows, drop_cols, parallel language for both parameters. I'm concerned about this command, will ask group.

Reply:
Talked to Tim, we think it's best to change the name to `read_matrix_table` and remove the parameters.